presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -66,7 +68,7 @@ public void setUp()
         mockClient = new MockThriftMetastoreClient();
         ThriftHiveMetastore thriftHiveMetastore = createThriftHiveMetastore();
         ListeningExecutorService executor = listeningDecorator(newCachedThreadPool(daemonThreadsNamed("test-%s")));
-        metastore = new CachingHiveMetastore(
+        metastore = (CachingHiveMetastore) cachingHiveMetastore(
```

Comment:
It would be cleaner to make `cachingHiveMetastore` method return `CachingHiveMetastore`

Reply:
`cachingHiveMetastore` is just wrapping method. It will return delegate when caching is disabled.